docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -0,0 +1,266 @@
+---
+description: Administer swarm clusters with Docker EE for IBM Cloud
+keywords: ibm, ibm cloud, logging, iaas, tutorial
+title: Administer swarm clusters with Docker EE for IBM Cloud
+---
+
+Docker Enterprise Edition for IBM Cloud (Beta) comes with a variety of integrations that simplify the swarm administration process. Use the Docker EE for IBM Cloud CLI plug-in (`bx d4ic`) to quickly provision swarm mode clusters and resources. Manage your cluster with the `bx d4ic` plug-in and the Docker EE Universal Control Plane (UCP) browser-accessed interface.
+
+## Create swarms
+Create a Docker EE swarm cluster in IBM Cloud.
+
+> Your beta license allows you to provision up to 20 nodes
+>
+> For beta, your cluster can have a maximum of 20 nodes, up to 14 of which can be worker nodes. If you need more nodes than this, work with your Docker representative to acquire an additional Docker EE license.
+
+Before you begin:
+* [Complete the setup requirements](/docker-for-ibm-cloud/index.md).
+* Make sure that you have the appropriate [IBM Cloud infrastructure permissions](faqs.md).
```

Comment:
Do you want to link to a specific anchor within that file? If so, put `#anchor` after the `.md` in the link target.